sycl/test/basic_tests/version.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %RUN_ON_HOST %t.out
+
+#include <CL/sycl.hpp>
+#include <iostream>
+
+int main() {
+#ifndef __LIBSYCL_MAJOR_VERSION
```

Comment:
This only checks that it is defined, but it will not check if it is empty, which I think we want to check too